frontend/src/exams/generate-financial-report-modal.vue

```diff
@@ -87,6 +89,10 @@
               exam_type = 3
             }else if (this.selectedExamType === 'pesticide') {
               exam_type = 4
+            }else if (this.selectedExamType === 'all') {
```

Comment:
Is there a reason for translating the text into a number, and then filtering based on number in the API request as well? Seems like it's much more readable if we keep the everywhere

Reply:
just thought it was cleaner passing one number via url param then words

Reply:
I'll remove this block but I'll have to go through and re-test everything.

Reply:
pushed up